server/chat-commands/moderation.ts

```diff
@@ -793,7 +793,7 @@ export const commands: Chat.ChatCommands = {
 			);
 		}
 
-		this.addModAction(`${name} was banned ${week ? ' for a week' : ''} from ${room.title} by ${user.name}.${publicReason ? ` (${publicReason})` : ``}`);
+		this.addModAction(`${name} was banned${week ? ' for a week' : ''} from ${room.title} by ${user.name}.${publicReason ? ` (${publicReason})` : ``}`);
```

Comment:
I think it'd make for sense for the space to remain where it is, and just remove the extra space in ' for a week' instead.

Reply:
Oh I'm stupid, that doesn't work does it lol